rules/properties-order/README.md

```diff
@@ -49,6 +49,8 @@ Array of unprefixed property names or group objects. Within an order array, you
 		If `emptyLineBefore` specified, regardless of it's value, the first property in a rule would be forced to not have an empty line before it.
 
 		For `threshold`, refer to the [`emptyLineMinimumPropertyThreshold` documentation](#emptylineminimumpropertythreshold-number).
+		
+		If you extend [`stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard), be aware, that it enables the `declaration-empty-line-before` rule, which will interfere with the `emptyLineBefore` option of the `properties-order` rule. So you might need to disable the `declaration-empty-line-before` in order for the `properties-order` rule to work correctly.
```

Comment:
Let's make it a little bit more generic and not about `stylelint-config-standard` specifically. Something like “If this option is not working make sure you don't have `declaration-empty-line-before` configured in a conflicting way in your config or config you're extending (e. g. `stylelint-config-standard`)”.